components/ActualGamesList/ActualGame/ActualGame.tsx

```diff
@@ -22,9 +34,9 @@ export const ActualGame: React.FC<ActualGamePropsI> = ({
       </div>
       <div className={styles.proposer}>{proposer}</div>
       <div className={styles.rules}>
-        {rules.toLowerCase() === '0xc6f81d6610a0b1bcb8cc11d50602d490b7624a96'
+      {rules.toLowerCase() === ticTacToeAddress
           ? 'Tic-Tac-Toe'
-          : '0x6ede6f6f1aca5e7a3bdc403ea0ca9889e2095486'
+          : rules.toLowerCase() === checkersAddress
           ? 'Checkers'
           : rules}
```

Comment:
It would be better to only render the date here and run the logic somewhere outside.